eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -522,8 +549,8 @@ alonzoTxInfo pp lang ei sysS utxo tx = do
           { PV1.txInfoInputs = mapMaybe (uncurry txInfoIn) txIns
           , PV1.txInfoOutputs = mapMaybe txInfoOut (foldr (:) [] txOuts)
           , PV1.txInfoFee = transValue (inject @(MaryValue (EraCrypto era)) fee)
-          , PV1.txInfoMint = transMintValue (txBody ^. mintTxBodyL)
-          , PV1.txInfoDCert = foldr (\c ans -> transDCert c : ans) [] (txBody ^. certsTxBodyG)
+          , PV1.txInfoMint = transMultiAsset (txBody ^. mintTxBodyL)
```

Comment:
We need to switch it back to `transMintValue`. This is only fixed in with `PlutusV3`
```suggestion
          , PV1.txInfoMint = transMintValue (txBody ^. mintTxBodyL)
```